Core/src/org/sleuthkit/autopsy/modules/photoreccarver/PhotoRecCarverOutputParser.java

```diff
@@ -154,4 +154,4 @@
         List<LayoutFile> empty = Collections.emptyList();
         return empty;
     }
```

Comment:
We need a cancellation check in PhotoRecCarverOutputParser.parse, before processing each file object element.